bin/improver-regrid

```diff
@@ -0,0 +1,113 @@
+       cube."""
+
+    cli_specific_arguments = [(['source_data_filepath'],
+                               {'metavar': 'SOURCE_DATA',
+                                'help': ('A cube of data that is to be '
+                                         'regridded onto the target_grid.')}),
+                              (['target_grid_filepath'],
+                               {'metavar': 'TARGET_GRID',
+                                'help': ('A cube containing the grid to which '
+                                         'the source_data is to be regridded.'
+                                         )}),
+                              (['output_filepath'],
+                               {'metavar': 'OUTPUT_FILE',
+                                'help': ('The output path for the processed '
+                                         'NetCDF')}),
+                              (['--nearest'],
+                               {'default': False,
+                                'action': 'store_true',
+                                'help': ('If True, regridding will be '
+                                         'performed using iris.analysis.'
+                                         'Nearest() instead of Linear(). '
+                                         'Use for less continuous fields, e.g.'
+                                         ' precipitation.')}),
+                              (['--extrapolation_mode'],
+                               {'default': 'nanmask',
+                                'help': ('Mode to use for extrapolating data '
+                                         'in to regions beyond the limits of '
+                                         'the source_data domain. Modes are: '
+                                         'extrapolate - The extrapolation '
+                                         'points will take their value from '
+                                         'the nearest source point. '
+                                         'nan - The extrapolation points will '
+                                         'be be set to NaN. error - A '
+                                         'ValueError exception will be raised,'
+                                         ' notifying an attempt to extrapolate'
+                                         '. mask  - The extrapolation points '
+                                         'will always be masked, even if the '
+                                         'source data is not a MaskedArray. '
+                                         'nanmask - If the source data is a '
+                                         'MaskedArray the extrapolation points'
+                                         ' will be masked. Otherwise they will'
+                                         ' be set to NaN. Defaults to nanmask.'
+                                         )}),
+                              ]
+
+    cli_definition = {'central_arguments': [],
```

Comment:
Shouldn't we use the default central arguments ('input_file', 'output_file') instead of "source_data_filepath" and "output_filepath"?

Reply:
This does not allow for two input files in the order desired here, hence this implementation.